packaging/leapp-repository.spec

```diff
@@ -26,11 +23,29 @@ Repositories for leapp
 %package data
 License: Red Hat Enterprise Agreement
 Summary: Package evolution data for leapp
-Requires: %{name} = %{version}-%{release}
+Requires:   %{name} = %{version}-%{release}
 
 %description data
 Package evolution data for leapp.
 
+%package deps
+Summary:    Meta-package with system dependencies of %{name} package
+
+##################################################
+# Real requirements for the leapp-repository HERE
+##################################################
+Requires:   dnf >= 4
+%if 0%{?rhel} && 0%{?rhel} == 7
+# Required to gather system facts about SELinux
+Requires:   libselinux-python
+%else ## RHEL 8 dependencies ##
+# Requires:   systemd-container
```

Comment:
Why is this commented out?

Reply:
Because we do not need this RPM in our case